webpack.prod.config.js

```diff
@@ -50,9 +50,10 @@ baseConfig.plugins = [
       NODE_ENV: JSON.stringify('production'),
     },
   }),
-  new MiniCssExtractPlugin({ filename: '[name]-[hash].css', disable: false, allChunks: true }),
+  new MiniCssExtractPlugin({ filename: '[name]-[hash].css' }),
   new BundleTracker({
-    filename: './webpack-stats.json',
+    path: __dirname,
```

Comment:
Is this the right config for prod? Have you tested it with a deploy?

Reply:
I only tested the build locally, I'll test a deploy now